src/dcap.c

```diff
@@ -128,11 +128,15 @@ static int name_invalid(char *);
 static server * url2srv(dcap_url *);
 
 #ifndef HAVE_HTONLL
+#ifndef htonll
```

Comment:
Is it not a better option to write
```
#if !defined(HAVE_HTONLL) && !defined(htonll) 
```

Reply:
That sounds much better!
Thank you for telling me this.
Updated.